lectures/5-Tutorials/SymmetricTensors.md

```diff
@@ -176,27 +176,48 @@ ZZ : \mathbb C^2 \otimes \mathbb C^2 &\to \mathbb C^2 \otimes \mathbb C^2 : \\
 \end{align}
 ```
 
-If we denote the trivial $\mathbb{Z}_2$ irrep by $'0'$, corresponding to a local $\ket{+}$
-state, and the sign irrep by $'1'$, corresponding to a local $\ket{-}$ state, and recall
-that in this notation the fusion rules of $\mathbb{Z}_2$ are given by addition modulo 2, we
-can associate each of the above matrix elements to a so-called *fusion tree* of
-$\mathbb{Z}_2$ irreps with a corresponding coefficient of 1,
-
+We will denote the trivial $\mathbb{Z}_2$ irrep by $'0'$, corresponding to a local $\ket{+}$
+state, and the sign irrep by $'1'$, corresponding to a local $\ket{-}$ state. Given this
+identification, we can naturally associate the tensor product of basis vectors in the irrep
+basis to the tensor product of the corresponding $\mathbb{Z}_2$ irreps. One of the key
+questions of the [representation theory of groups](representation_theory) is how the tensor
+product of two irreps can be decomposed into a direct sum of irreps. This decomposition is
+encoded in what are often called the
+[*fusion rules*](https://en.wikipedia.org/wiki/Fusion_rules),
+```{math}
+a \otimes b \cong \bigoplus_c N_c^{ab} c,
+```
+where $N_{ab}^c$ encodes the number of times the irrep $c$ occurs in the tensor product of
+irreps $a$ and $b$. These fusion rules are called *Abelian* if the tensor product of any two
+irreps corresponds to exactly one irrep. This is the case for the $\mathbb{Z}_2$ irreps,
+whose fusion rules are given by addition modulo 2,
+```{math}
+0 \otimes 0 \cong 0, \quad 0 \otimes 1 \cong 1, \quad 1 \otimes 0 \cong 1, \quad 1 \otimes 1 \cong 0.
+```
```

Comment:
Misschien nog explicieter:

Z2 acts trivially on the + state, while it yields a minus sign for the - state, thus leading to respectively the trivial and sign irrep. 
[...]
We can read off the fusion rules by considering the action of Z2 on the tensor product of states, and we find that + \otimes + and - \otimes - now transform trivially, while + \otimes - and - \otimes + transform under the sign representation. In other words, the fusion rules are given by addition modulo 2,
[...]